api.md

```diff
@@ -469,7 +535,74 @@ CREATE INDEX ON conditions(time, location) USING brin
 ```
 
 ---
+## detach_data_node() [](detach_data_node)
+
+Detach a data node from one hypertable or from all hypertables.
 
+This function is typically used if a data node should no longer be
+used by a hypertable. Then it need to be removed from all hypertables
+that use it.
+
+Another reason for detaching a data node from a hyper table is that
+you want to have fewer partitions for that hypertable compared to
+other hypertables in the database.
+
```

Comment:
Clarify that this function should be executed at the access node?

Reply:
Done